reference/composedshader.md

```diff
@@ -0,0 +1,105 @@
+## ComposedShader
+
+```
+ComposedShader {
+  vrmlField MFNode fields []
+  vrmlField MFNode parts  []
+}
+```
+
+### Description
+
+The [ComposedShader](#composedshader) node defines a shader as a list of vertex or fragment programs and the fields passed to these programs.
+
+The `parts` type is a list of [ShaderPart](shaderpart.md) nodes.
+Generally a [ShaderPart](shaderpart.md) node of type "VERTEX" and another of type "FRAGMENT" are expected.
+If several [ShaderPart](shaderpart.md) node of the same type are given, then only the last ones are used.
```

Comment:
Usually in other nodes where only one value is used but multiple values can be inserted, the first one is the active one and not the last one, like in ImageTexture URL, Led lights etc.
Why did you decide to make the last one the active one?